.github/workflows/docs-build.yml

```diff
@@ -36,3 +36,52 @@ jobs:
           git add -f docs static/statuses.json static/arguments.yml static/classes.yml
           git commit -m "docs: build docs" || true
           git push
+      - name: Set up node js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 15
+      - name: Run docs
+        run: |
+          cd docs
+          yarn
+          yarn develop & npx wait-on http://localhost:8000
+      - name: Run axe checks on examples of component docs
+        run: |
+          npm install -g @axe-core/cli
+          files=($(grep -r -l "Examples" docs/content/components))
+          declare -a needs_to_be_fixed=(
+            autocomplete 
+            autocompleteitem
+            dropdown
+            dropdownmenu
+            imagecrop
+            markdown
+            navigationtab
+          )
+          
+          echo "############################################################"
+          echo "WARNING"
+          echo "The following components will be excluded from this automated accessibility check."
+          echo "They contain known violations and should be addressed:"
+          printf '%s\n' "${needs_to_be_fixed[@]}"
+          echo "############################################################"
+
+          declare -a doc_urls
+          for i in "${!files[@]}"; 
+          do
+            filename=$(basename -- "${files[$i]}")
+            component="${filename%.*}"
+            if [[ " ${needs_to_be_fixed[@]} " =~ " ${component} " ]]
+            then
+              continue
+            else
+              doc_urls+=("localhost:8000/components/${component}")
+            fi
+          done
+          printf -v joined '%s, ' "${doc_urls[@]}"
+          axe $joined -i .frame-example --save docs-example-accessibility.json --disable color-contrast --exit
```

Comment:
Followup TODO:

Pull this out into a script. 
Need to look into how to run script in github actions.

Reply:
If we pull this into its own script, we can even re-write this in something like ruby or node since bash debugging and developing can become a bit unwieldy 😬